plugins/altstatusbar.koplugin/main.lua

```diff
@@ -0,0 +1,271 @@
+-- plugin for configuration of the Alt Status Bar
+
+--[[
+from https://github.com/koreader/koreader/issues/5848#issuecomment-584682914
+thanks to @poire-z
+That top status bar is not managed by KOReader, but by crengine (the engine thats renders EPUB files).
+You can tweak a bit of what it will show/not show by editing koreader/data/cr3.ini, these bits should apply to it:
+
+crengine.page.header.chapter.marks=1
+crengine.page.header.font.color=0xFF000000
+crengine.page.header.font.face=Noto Sans
+crengine.page.header.font.size=22
+
+window.status.battery=1
+window.status.battery.percent=0
+window.status.clock=0
+window.status.line=0
+window.status.pos.page.count=1
+window.status.pos.page.number=1
+window.status.pos.percent=0
+window.status.title=1
+]]
+
+--if true then return { disabled = true, } end
+
+local Device = require("device")
+local Event = require("ui/event")
+local ConfirmBox = require("ui/widget/confirmbox")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local T = require("ffi/util").template
+local logger = require("logger")
+local _ = require("gettext")
+
+local AltStatusBar = WidgetContainer:new{
+    name = "altstatusbar",
+    is_doc_only = true,
+}
+
+function AltStatusBar:isEnabled()
+    return G_reader_settings:readSetting("copt_status_line") == 0
+end
+
+function AltStatusBar:onReadSettings()
```

Comment:
onReadSettings(config) accepts an argument, which will be the docsettings (per document settings).
copt_status_line is per-document (for the reason I explained earlier).
So, your self.enabled should come from it, not just from G_reader_settings.
I guess this should be done like in there:
https://github.com/koreader/koreader/blob/eb66856d810d5f4b292ffa860960d898f4a03d0f/frontend/apps/reader/modules/readertypeset.lua#L24-L26

Oh, just like it's done there :) 
https://github.com/koreader/koreader/blob/eb66856d810d5f4b292ffa860960d898f4a03d0f/frontend/apps/reader/modules/readercoptlistener.lua#L6-L21

Reply:
No. All menu entries apply the global settings, so these can only be changed if the AltStatusBar is globally enabled.
But I can drop the `enabled_func` at all, than the configuration can be changed, even the AltStatusBar is disabled. But it may be confusing to switch some entries and nothing happens.

Only the `status_line` comes from docsettings. This is stated in the menu with `...for this document`.